src/integration/java/com/smartcar/sdk/VehicleTest.java

```diff
@@ -0,0 +1,83 @@
+package com.smartcar.sdk;
+
+import com.smartcar.sdk.data.SmartcarResponse;
+import com.smartcar.sdk.data.VehicleInfo;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+/**
+ * Integration Test Suite: /vehicles/:id
+ */
+public class VehicleTest extends IntegrationTest {
+    private Vehicle vehicle;
+
+    @BeforeSuite
+    public void beforeSuite() throws Exception {
+        String[] vehicleIds = AuthClient.getVehicleIds(this.getAuth().getAccessToken()).getData().getVehicleIds();
+        this.vehicle = new Vehicle(vehicleIds[0], this.getAuth().getAccessToken());
+    }
+
+    /**
+     * Tests that vehicle info can be obtained.
+     */
+    @Test
+    public void testInfo() throws SmartcarException {
+        VehicleInfo info = this.vehicle.info();
```

Comment:
Just so I know how this test works, do these tests actually hit our servers?

Reply:
Correct. The goal with these tests are essentially that -> to exercise the SDK under the same circumstances a developer would.